features/0453-issue-credential-v2/README.md

```diff
@@ -146,8 +146,8 @@ Description of attributes:
 
 Credential Format | Format Value | Link to Attachment Format | Comment |
 --- | --- | --- | --- | 
-DIF Credential Manifest | `dif/credential-manifest@v1.0` | [`propose-credential` attachment format](../0511-dif-cred-manifest-attach/README.md#propose-credential-attachment-format) | 
-Hyperledger Indy Credential Abstract | `hlindy/cred-abstract@v2.0` | [`cred abstract` format](../0592-indy-attachments/README.md#cred-abstract-format)|
+DIF Credential Manifest | `dif/cred-manifest@v1.0` | [`propose-credential` attachment format](../0511-dif-cred-manifest-attach/README.md#propose-credential-attachment-format) | 
```

Comment:
I agree. I don't know what this change is trying to fix.

Reply:
The identifier in `Format Value` is meant to match the one in the respective attachment format RFC. Changing this here to `dif/cred-manifest@v1.0` without an equal change in RFC0510 creates misalignment.

But still - what are we trying to solve with this change?

Reply:
I originally implemented the standard with its hlindy-zkp@v1.0 everywhere [https://github.com/hyperledger/aries-rfcs/tree/de0092a41b7d54c4fe58a69e0dba24ae8bec9360/features/0454-present-proof-v2]. @TimoGlastra  branched off that implementation for DIF credentials. Then the practice of the standard changed and I missed it - so I would ask the same question against that change in the first place: it requires a redesign of the format abstraction which encapsulates a bunch of logic, and then that will make Timo's merge into their DIF implementation all the more laborious.

PR 607 suggests that a format identifier should identify a format (as it did in the first place), not a protocol and attachment type - by construction it is already embedded in the protocol and message, allowing for only one attachment type per format. However, if the consensus is otherwise, I can go back and change things accordingly.

In any case, the effort level is likely to be man-days at most, not weeks or months.

Reply:
@llorllale @dhh1128 What we're trying to solve here is what I described in this issue: #599. We didn't get to it during the WG call. 

The specific format identifiers feels like it adds unnecessary complexity. However, I think Daniel raises a good point that the formats could be used outside of the issue credential / present proof protocol. I had not thought about it that way.